### DISCUSSION TITLE
I don't receive the "magic link" email

### DISCUSSION THREAD

**Discussion by omejace** [mozilla/hubs#5725] [Hubs Cloud Questions] [answered]
Hi!
I'm deploying an environment using "Hubs Cloud Enterprise".
However, after a successful deployment, I don't receive the "magic link" email to my "Administrator Email Address",
 and when I go to the Sine-Up page and enter another valid email address, I don't receive the email.

**What I have tried**
- Deploying a Stack with "N. Virginia (us-east-1) "
- Unlock the "AWS SES Sandbox"
- Verify email addresses

Also, the following error message is output in the developer console of the browser.

`
Uncaught (in promise) TypeError: NetworkError when attempting to fetch resource. hub-10e0507b6f270aa37347.js:1:84974
Firefox can't establish a connection to the server at wss://eloquent-troll.example.io/socket/websocket?vsn=2.0.0. store-b95640929c98db988ce5.js:8:28959
Socket error, changed endpoint to wss://eloquent-troll.example.io/socket/websocket hub-10e0507b6f270aa37347.js:1:1412489
`

Any ideas to solve this?

**Comment by hrithikwins** [comment, marked as answer]
https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#youre-in-the-aws-sandbox-and-people-dont-receive-magic-link-emails

You'll need to do additional setup to receive magic links from your cloud server
